### PR TITLE
ashley/add support for conversion of cvss vectors to jira comments

### PR DESCRIPTION
**Pull request by ashleyblackmore**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -15,6 +15,9 @@ classifier =
           Programming Language :: Python :: 3.4
           License :: OSI Approved :: BSD License
 
+[pycodestyle]
+max-line-length = 100
```
Comment (dbaxa) [inline setup.cfg]: I do like to keep to 80. /me shurgs.

File: cvss_util/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "2.0.2"
+__version__ = "2.3.0"
```
Comment (dbaxa) [inline cvss_util/__init__.py]: Hmm we can probably delete this now - as we are using pbr.

**Review by dbaxa — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (mark-adams) [reply setup.cfg]: Bah... who only has 80 lines nowadays? :-)

**Review by mark-adams — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (dbaxa) [reply setup.cfg]: That's not the point :-) .

File: cvss_util/utils/test.py [hunk 3]
```diff
@@ -1,9 +1,73 @@
...
+|| IntegImpact | Partial |
+|| AvailImpact | Complete |
+
+"""
+        self.assertEqual(
+            create_cvss_format.cvss_v2_vector_to_jira_table(
+                vector_string,
+                9.0,
+            ),
+            expected
```
Comment (dbaxa) [inline cvss_util/utils/test.py]: @ashleyblackmore what you could do is test parsing the generated output with the comment parsing code.

File: setup.cfg (hunk 1 above)
Comment (ashleyblackmore) [reply setup.cfg]: I'm happy to set it back to 80, but often in python it feels like a trade-off between line length, keeping within pep8 multiline style, and adequately named members/functions. In particular, I feel like there isn't a nice way to style function declarations across multiple lines (whereas function calls are somehow fine), so I chose to bump this instead. :D

File: cvss_util/__init__.py (hunk 2 above)
Comment (ashleyblackmore) [reply cvss_util/__init__.py]: I get this upon running setup.py once removed: `package init file 'cvss_util/__init__.py' not found (or not a regular file)` – what would you do here?

Comment (ashleyblackmore) [reply cvss_util/__init__.py]: Something like this?

```
import pbr.version


__version__ = pbr.version.VersionInfo('cvss_util').version_string()
```

**Pull request merged**